examples/hubble/hubble-cli.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hubble-cli
+  labels:
+    app.kubernetes.io/name: hubble-cli
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: hubble-cli
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: hubble-cli
+    spec:
+      containers:
+      - name: hubble-cli
+        image: quay.io/cilium/hubble:v0.8.0
```

Comment:
optional nit:

I wonder if it made sense to add an additional `sed` command in `images/scripts/update-hubble-version.sh` to bump this line whenever we bump the Hubble CLI inside Cilium.

Reply:
I share this concern and what is proposed here makes sense to me.

Reply:
Good catch! Opened a new PR since this one has already been merged, see https://github.com/cilium/cilium/pull/16537